packages/extension/src/hooks/useTransactionStatus/useTransactionStatus.tsx

```diff
@@ -134,15 +133,6 @@ export const useTransactionStatus = ({
           />
         );
       }
-      Sentry.captureException('Transaction failed - errorFees: ' + errorFees);
```

Comment:
Why did you remove this?

Reply:
I have changed the way the fees calculation errors are handled (better) and display (smoother). Also it's not relevant to track this with Sentry, especially since the fees calculation hook can throw errors when the hook has race conditions (see the comments in the code)